app/Http/Controllers/UserController.php

```diff
@@ -44,6 +45,13 @@ public function destroy($id){
   }
 
   public function update(User $user){
+    $pw = $this->request->only('originalPassword')['originalPassword'];
+
+    // if the password doesn't match and the user is not an admin this person should not be updating shit!
```

Comment:
This comment is addressed by the policies that have yet to be merged into master, don't worry ;)